static/app/components/panels/panelHeader.tsx

```diff
@@ -33,8 +33,7 @@ const PanelHeader = styled('div')<Props>`
   font-weight: ${p => p.theme.fontWeightBold};
   text-transform: uppercase;
   border-bottom: 1px solid ${p => p.theme.border};
-  border-radius: calc(${p => p.theme.borderRadius} - 1px)
-    calc(${p => p.theme.borderRadius} - 1px) 0 0;
+  border-radius: calc(${p => p.theme.borderRadius}) calc(${p => p.theme.borderRadius}) 0 0;
```

Comment:
the radius before was 6px - 1. This is now 4px. I think we want 5px;

4px
![image](https://github.com/user-attachments/assets/b7d43f32-8d82-466e-91cf-52262e90a020)

5px
![image](https://github.com/user-attachments/assets/38f75da0-c4d1-4137-8008-651423f7f1e4)

Reply:
We have a rogue provider changing this, let me do +1 and remove it in a followup PR